newsroom/templates/search_tips_regular.fy_ca.html

```diff
@@ -0,0 +1,25 @@
+<div class="tips-content">
+    <div>
+        <p><b>Yleistä Mediapankin hauista</b></p>
+        <p>Mediapankki käyttää boolean-hakua, jossa erilaisia komentoja ja hakutermejä yhdistelemällä voi hakea juttujen otsikoista, leipätekstistä ja metatiedoista.</p>
+
+        <p>Jos hakukenttään kirjoittaa useamman kuin yhden sanan tai hakutermin, hakutuloksina esitetään jutut, joissa ne kaikki esiintyvät.</p>
+
+        <p>Mediapankin haku on hyvin tarkka siitä, että merkit ovat oikeanlaisia, joten jos haku ei anna odotettuja tuloksia, kokeile uudestaan näiden ohjeiden avulla. Esimerkiksi lainausmerkkien pitää olla juuri oikeanlaiset eli kuten tässä ohjeessa.</p>
+    </div>
+    <hr />
+    <div>
+        <p><b>Tärkeimmät hakutermit</b></p>
+        <p>* = katkaisee sanan, jotta voidaan ottaa huomioon taivutusmuodot. Esimerkiksi älä hae korhonen vaan korho*. Voit käyttää myös tähteä sanan sisällä. Jos esimerkiksi et muista, oliko haettava henkilö Silvennoinen vai Silventoinen, voit hakea Silven*oi*</p>
+
+        <p>"" = lainausmerkkien avulla voit tehdä kahden tai useamman sanan yhdistelmästä täsmällisen termin jota haetaan. Esimerkiksi voit hakea "eduskunnan kahvila", ei eduskunnan kahvila. Huom! Tähtisymboli ei toimi sitaattimerkkien sisällä.</p>
+
+        <p>AND tarkoittaa, että kaikkien hakutermien pitää olla haussa mukana. Monesti AND on tarpeeton Mediapankissa, koska välilyönti vastaa AND-termiä, mutta pitkissä ja monimutkaisissa hakulausekkeissa AND voi auttaa hahmottamaan, mitä oikeastaan on hakemassa.</p>
+
+        <p>OR tarkoittaa, että hakutulokseksi päätymiseen riittää, että joku hakutermeistä osuu.</p>
+
+        <p>NOT tarkoittaa, että jutut joissa on tämä hakutermi, eivät tule mukaan hakutuloksiin.</p>
+
+        <p>Lisää hakuohjeita: <a href="https://stt.fi/ajankohtaista-tietoa-media-asiakkaille/stt-mediapankki/">https://stt.fi/ajankohtaista-tietoa-media-asiakkaille/stt-mediapankki/</a></p>
```

Comment:
I think we don't need the full link as a text, you can remove that `https://` inside `a`